easybuild/framework/easyblock.py

```diff
@@ -158,12 +158,17 @@ def __init__(self, ec):
         # module generator
         self.module_generator = module_generator(self, fake=True)
 
-        # modules footer
+        # modules footer/header
         self.modules_footer = None
         modules_footer_path = build_option('modules_footer')
         if modules_footer_path is not None:
             self.modules_footer = read_file(modules_footer_path)
 
+        self.modules_header = None
+        modules_header_path = build_option('modules_header')
+        if modules_header_path is not None:
```

Comment:
the default value is `None`; if it's empty, you're doing something weird (e.g. use `eb --modules-header=''`, and you deserve to know you did something wrong?

You will get a proper error via `read_file` in that case, which is certainly better than just ignoring this and leaving people wondering why it doesn't work.